### DISCUSSION TITLE
Is this a small mistake in the example of carla data convertion given?

### DISCUSSION THREAD

**Discussion by yalezhang2021** [carla-simulator/carla#4134] [Q&A]
In this page https://carla.readthedocs.io/en/latest/ref_sensors/#radar-sensor there is an example of radar data's convertion.
# To get a numpy [[vel, altitude, azimuth, depth],...[,,,]]:
points = np.frombuffer(radar_data.raw_data, dtype=np.dtype('f4'))
points = np.reshape(points, (len(radar_data), 4))

I think in the notation it should be: # To get a numpy [[vel, azimuth, altitude, depth],...[,,,]]:

The positions of azimuth and altitude are reversed.

**Comment by corkyw10** [comment]
Hi @yalezhang2021, you're right, thank you for pointing it out. I will get it fixed in the next version of the docs
